lua/ulx/modules/sh/pcrestrict.lua

```diff
@@ -0,0 +1,45 @@
+CATEGORY_NAME = "CFC"
+
+local function addToWhitelist(callingPlayer, targetPlayers)
+    local affected_plys = {}
+
+    for _, ply in pairs(targetPlayers) do
+        local isPlyWhitelisted = isPlayerWhitelisted(ply)
+
+        if isPlyWhitelisted then 
+            ULib.tsayError(callingPlayer, ply:Name().." is already whitelisted!", true)
+        else
+            table.insert(affected_plys, ply)
+        end
+    end
+
+    addPlayerToPropcoreWhitelist(affected_plys)
+    ulx.fancyLogAdmin(callingPlayer, true, "#A added #T to the propcore whitelist", affected_plys)
+end
+
+local PCWhitelist = ulx.command(CATEGORY_NAME, "ulx addwhitelist", addToWhitelist, "!addwhitelist")
```

Comment:
These commands should have better names, `ulx allowpropcore` and `ulx denypropcore` or something